Sources/RedCat/ComposedStore.swift

```diff
@@ -1,115 +0,0 @@
-//
-//  File.swift
-//  
-//
-//  Created by Данил Войдилов on 30.05.2021.
-//
-
-import Foundation
-
-public final class ComposedStore<First: StoreProtocol, Second: StoreProtocol, State>: Store<State> {
-	
-	public let first: First
-	public let second: Second
-	public let getState: (First.State, Second.State) -> State
-	
-	public override var state: State {
-		getState(first.state, second.state)
-	}
-	
-	public init(_ first: First, _ second: Second, state: @escaping (First.State, Second.State) -> State) {
-		self.first = first
-		self.second = second
-		self.getState = state
-	}
-	
-	override public func send<Action>(_ action: Action) where Action : ActionProtocol {
-		first.send(action)
-		second.send(action)
-	}
-	
```

Comment:
This is gone now, but for completeness sake (because I remembered seeing it and remembered that I didn't address this but couldn't find it) this ```&&``` should have been an ```||``` I guess.